bin/lerna-docker

```diff
@@ -41,7 +41,9 @@ sleep 1
 REGISTRY_PID=$!
 npm-cli-adduser --registry http://localhost:4873 --username lerna --password lerna --email lerna@example.org
 npx lerna exec -- npm publish --silent --registry http://localhost:4873 > /dev/null
-if [ "$(uname)" == "Darwin" ]; then
+
+# If we run on mac or WSL we use docker internal
+if [[ $(grep -i Microsoft /proc/version) ]] || ["$(uname)" == "Darwin"]; then
```

Comment:
`/proc/version` doesn't exist on all systems (like mine), so this will cause the script to error.
We should probably add an additional check first for the existence of this file.

Reply:
Right, I added the check, let me know if it looks ok.

Reply:
Are you sure the scope of the file-check is correct? So that the &&-condition also doesn't force this file to exist on Mac before passing the if-condition?
(I'm not sure myself, which is why I'm asking)
If we want to make sure, we could wrap in `()` to clarify the scope.

Reply:
Not 100% sure, so wrapping in () is indeed better, should be ok now